languages/ruby/lib/oso/errors.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Oso
+  class Error < StandardError
+  end
+
+  class AuthorizationError < StandardError
+  end
```

Comment:
I think it's pretty standard to extend `StandardError` instead of `RuntimeError`, but I don't have a strong opinion so I'll go with `RuntimeError` for consistency